### DISCUSSION TITLE
[FR] <title>Package not downloading

### DISCUSSION THREAD

**Discussion by shahdnizar** [cjbarrie/academictwitteR#334] [General]
### Describe the solution you'd like

Hi, I am trying to install the package using the following command "devtools::install_github("cjbarrie/academictwitteR", build_vignettes = TRUE)" but it fails to download. The message I get is "Error in utils::download.file(url, path, method = method, quiet = quiet,  :  download from 'https://api.github.com/repos/cjbarrie/academictwitteR/tarball/HEAD' failed". Any idea what might be the problem and how to resolve this?

**Comment by chainsawriot** [comment]
@shahdnizar Check your internet connection and try again. Or use the CRAN version.
